independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/EndpointIndexer.java

```diff
@@ -627,7 +627,15 @@ private static boolean moreThanOne(AnnotationInstance... annotations) {
         if (annotation == null) {
             return null;
         }
-        String[] originalStrings = annotation.value().asStringArray();
+
+        String[] originalStrings;
+        AnnotationValue value = annotation.value();
+        if (value == null) {
+            originalStrings = new String[] { MediaType.WILDCARD };
```

Comment:
Don't we want JSON by default if around? Won't this change defeat that?

Reply:
TBH I'm not sure what the semantics should be if the user specifies a `@Produces` with no value. Spec says wildcard. I say it's useless.

Perhaps we should just treat this as if the annotation was not there?

Or perhaps it's useful to override a class-level `@Produces(text)` annotation?

Reply:
I say that if it's thereς with no default value we keep the spec behavior.
It's pretty stupid, but it the user set it explicitly, the maybe they know what they are doing?

Reply:
That's one way to look at it. Another would be that if it's original purpose was to override the class annotation back to its "default" behaviour, we should make this match our own "default" and consider this as having no value and not looking at the class annotation.

But frankly this is so edge-case that I doubt it's worth it.

Reply:
OK, works for me if it's only when you have empty annotations.